modules/47_regipol/regiCarbonPrice/declarations.gms

```diff
@@ -57,6 +57,7 @@ Parameter
   p47_firstTargetYear(ext_regi)                      "first year with a pre defined policy emission target in the region [year]"
   p47_lastTargetYear(ext_regi)                       "last year with a pre defined policy emission target in the region [year]"
   p47_currentConvergencePeriod(ext_regi)             "auxiliar parameter to store the current target year being executed by the convergence algorithm [year]"
+  p47_currentConvergence_iter(iteration,ttot,ext_regi) "auxiliar parameter to store if emission target is being executed by the convergence algorithm in the specific iteration [0 or 1]"
```

Comment:
a) I don't fully get the description. 

would "auxiliary parameter to store if carbon price adjustment in response to a specific emission target is being executed by the convergence algorithm in the specific REMIND iteration [0 or 1]" also describe it? then I would prefer something along those lines. 

b) does this describe whether ANY price adjustment is being done, or whether or not a price adjustment is done due to one specific emission target? (so something like "there is an adjustment due to the ETS target for 2030, but no adjustment due to the ESR target for 2030"). in that case, the parameter would need to contain the emiMktExt as well, right?

only asking - if this would complicate things maybe it doesn't matter

Reply:
thanks!